win/win-tts/Program.cs

```diff
@@ -12,12 +15,44 @@ static void Main(string[] args)
             var synth = new SpeechSynthesizer();
             var stream = new MemoryStream();
             var format = new SpeechAudioFormatInfo(16000, AudioBitsPerSample.Sixteen, AudioChannel.Mono);
+            SelectVoice(synth);
             synth.SetOutputToAudioStream(stream, format);
             synth.SpeakSsml(args[0]);
 
             var stdout = Console.OpenStandardOutput();
             stream.Position = 0;
             stream.CopyTo(stdout);
         }
+
+        private static void SelectVoice(SpeechSynthesizer synth)
+        {
+            StringBuilder sb = new StringBuilder();
+            string voice = ConfigurationManager.AppSettings.Get("voice");
```

Comment:
I think it would be cool if we could do e.g. `ATIS Kutaisi 251.000, VOICE WIN:Hazel` to select the voice that should be used.

Wo could therefore update the program to receive multiple command line arguments, where one of them is the voice.

The voice could simply be `args[1]` or we need a more sophisticated approach to allow us to do something like `win-tts.exe --voice Haze --text "...."`.

The part that sets the voice when calling `win-tts.exe` doesn't need to be part of this PR, I could do the Rust part of it.

Reply:
That's a nice idea - having a different voice for different airbases. However, you would need the information of what voices are actually installed on the system and can be chosen from.

As for the program arguments: I suggest to keep it simple for now. First arg[0] is the text, second (optional argument is the voice name. I suggest using the full name for the time being (e.g. "windows haze desktop") as I'm note sure if all voices have this exact naming pattern. I'll try to find out what additional voices can be installed.

Updated PR incoming.